src/auth/auth.service.ts

```diff
@@ -0,0 +1,61 @@
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+
+import { parseApiKeysConfig } from '~/auth/auth.utils';
+import { CreateApiKeyDto } from '~/auth/dto/create-api-key.dto';
+import { DeleteApiKeyDto } from '~/auth/dto/delete-api-key.dto';
+import { ApiKeyModel } from '~/auth/models/api-key.model';
+import { generateBase64Secret } from '~/common/utils';
+import { User, UserId } from '~/users/user.consts';
+
+@Injectable()
+export class AuthService {
+  private userStore: Record<UserId, User> = {};
```

Comment:
Any specific reason around `Record<UserId, User>`? Why not directly make it `Record<UserId, string[]>` to map api keys to userId?